### PR TITLE
Fixed some VTK error messages that showed up when ending configure

### PR DESCRIPTION
**Pull request by chaosphere2112**
Should hopefully fix @williams13's VTK error messages; managed to reproduce them locally and eliminated them. They cropped up when closing the configure menu when a bunch of plots were open.

### CODE REVIEW

File: Packages/vcs/Lib/vtk_ui/manager.py [hunk 1]
```diff
@@ -78,6 +78,7 @@ def detach(self):
             w.detach()
         if self.window.HasRenderer(self.renderer):
             self.window.RemoveRenderer(self.renderer)
+        self.renderer.RemoveAllViewProps()
```
Comment (aashish24) [inline Packages/vcs/Lib/vtk_ui/manager.py]: You shouldn't need this but if you do then you want to set self.renderer = None since at this point I think it won't be of further use.

File: Packages/vcs/Lib/configurator.py [hunk 2]
```diff
@@ -292,8 +292,6 @@ def detach(self):
         # if all of the widgets have been cleaned up correctly, this will delete the manager
         vtk_ui.manager.delete_manager(self.interactor)
 
-        self.interactor.Render()
```
Comment (aashish24) [inline Packages/vcs/Lib/configurator.py]: +1

File: Packages/vcs/Lib/vtk_ui/manager.py (hunk 1 above)
Comment (chaosphere2112) [inline Packages/vcs/Lib/vtk_ui/manager.py]: Yeah, I basically only added this line as backup in case I screwed something up elsewhere; the "detach" functions are used throughout my `vtk_ui` package to make elements completely remove themselves from the VTK render stack and make them garbage collectable. I'll go ahead and set renderer to `None` here.
